build-tools/automation/azure-pipelines.yaml

```diff
@@ -1368,6 +1369,55 @@ stages:
     msiNupkgArtifactName: vs-msi-nugets
     condition: eq(variables['MicroBuildSignType'], 'Real')
 
+- stage: push_nugets_to_nuget_org
+  displayName: Push to NuGet.org
+  dependsOn: dotnet_prepare_release
+  condition: and(eq(variables['MicroBuildSignType'], 'Real'), eq(dependencies.dotnet_prepare_release.result, 'Succeeded'))
+  jobs:
+  - job: wait_for_nuget_org_approval
+    displayName: Wait For NuGet.org Approval
+    timeoutInMinutes: 4320 # The job timeout should be longer than the approval task timeout.
+    pool: server
+    steps:
+    - task: ManualValidation@0
+      timeoutInMinutes: 2880 # Approval times out in 2 days, but the stage can be re-ran if needed.
```

Comment:
I've been wanting to change this with the other manual validation task, I think a two day waiting period is too long. Maybe we can change this to be 60 minutes for the job and 30 minutes for the task since we can re-run it as many times as we want.